subprojects/core-api/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -196,6 +197,14 @@ default void includeFlat(String... projectNames) {
      */
     Settings getSettings();
 
+    /**
+     * Provides access to various important directories for this settings object.
```

Comment:
💅  I find the wording rather artificial, especially the `settings object` part, because this is a method on the setting object.

```suggestion
     * Provides access to settings-related directories and allows resolving relative files.
```

Reply:
I borrowed that from Project's javadoc. With the paragraph on relative locations, hopefully is better.